### ISSUE TITLE
Add Lato v2.0

### ISSUE THREAD

**Issue by eternal-sorrow** [google/fonts#6]
Since 2.0 version Lato has started supporting cyrillic symbols along with some other languages. So, please add new version of this nice font.

**Comment by rsheeter** [comment]
Ty for the heads up, will investigate.

For my own reference, http://www.latofonts.com/2014/02/27/lato-2-0-released/.

**Comment by codeman38** [comment]
Lato 2.0 was tested a while back but rejected due to bad hinting that caused glitchy rendering on Windows. However, the hinting bug has since been fixed upstream: http://www.latofonts.com/2014/09/03/lato-family-updated-version-2-010/

**Comment by davelab6** [comment]
@twardoch has promised to publish the sources on Github eventually :)

**Comment by twardoch** [comment]
Yes, I will. But they're in a VERY chaotic state right now due to the complexity of the project. Mixture of FontLab Studio 5, manual stuff, hardcoded Python snd some unpublishable 3rd party tools. I'm the only person who can build them right now but I definitely plan to upgrade them to a more portable form this summer.

**Comment by nqst** [comment]
Is there any news on this issue?

**Comment by akhatskevich** [comment]
Need cyrillic symbols of Lato so much! So, is there any progress?

**Comment by simonsarris** [comment]
Happy issue anniversary, any word on getting Lato 2 support?

**Comment by smilemd** [comment]
up

**Comment by denniskuligin** [comment]
up

**Comment by davelab6** [comment]
Soon!

**Comment by martinzubek** [comment]
Hey guys, anything new on this?

**Comment by random-stranger** [comment]
What's happening with this one, why it is so hard to do? [Early 2016](https://github.com/google/fonts/issues/165#issuecomment-166634331), [soon in June 2016](https://github.com/google/fonts/issues/6#issuecomment-224765579). It's almost December, the **end** of the year.

Do people in Google use another understanding of time? Lato 2.0 [was released](http://www.latofonts.com/2014/02/27/lato-2-0-released/) in February, 2014, almost **3 years ago**. It's 1000 days ago. Is it so hard to finally dedicate a day or two and update the font?

@davelab6 Can you finally keep your promise and update this font at least in 2016?

cc @m4rc1e, @alexeiva can someone help him as this promises can take forever. ?

**Comment by fitojb** [comment]
@random-stranger Do any of them owe you anything at all, so that you're entitled to make such exigences? What is stopping you from simply self-hosting the font if you need it so desperately?

**Comment by davelab6** [comment]
@fitojb to be fair, some places require Google Fonts API to be used for their own reasons. 

@random-stranger when I pushed this the hinting had a problem, so I need to verify the hinting from this vendor is good in the latest release, and I didn't do this. However, @m4rc1e may be able to check this for us. 

I can't commit to a specific date.

**Comment by m4rc1e** [comment]
I'll look into this today, if they pass our QA, I'll send a pr. However, it will take more time to get them served from the api.

As @fitojb says, self hosting is not a bad option for the time being.

Cheers,
Marc

**Comment by twardoch** [comment]
I haven't heard any reports of Lato 2.015 failing on screen rendering, but I did hear of printing problems, even with the most current version, which most likely are related to hinting and `ttfautohint` — but it's next to impossible to debug those problems.

**Comment by m4rc1e** [comment]
Alright,

We have some minor issues and 2 relatively serious ones.

## Serious
### [FB](https://github.com/googlefonts/fontbakery) report for [v2 binaries](http://www.latofonts.com/lato-free-fonts/#download)

**All Weights**
- "ERROR: Glyph ["uni200C"](http://www.fileformat.info/info/unicode/char/200c/index.htm) has ink. It needs to be replaced by an empty glyph",
            "ERROR: Glyph ["uni200D"](http://unicode.org/cldr/utility/character.jsp?a=200D) has ink. It needs to be replaced by an empty glyph"
Lato V2 has:
![screen shot 2016-11-21 at 08 11 11](https://cloud.githubusercontent.com/assets/7525512/20473606/1e912fd0-afc2-11e6-9c38-a8a625b31d9b.png)

**Some weights**
- "ERROR: OS/2 usWinAscent & usWinDescent Changes that must be applied to this font: OS/2 usWinAscent from 1974 to 2109 | OS/2 usWinDescent from 426 to 527".

This will cause clipping on glyphs which ascend past 1974 units, same with the descending glyphs. I Recently found this issue on [Overpass](https://github.com/RedHatBrand/Overpass/issues/33). However, the metrics do match the previous release. This means it existed in the previous version which made it into fonts.google.com

## Minor
- Family needs splitting, due to api font name constraints. Thin and Heavy will need their own separate families.

- Licensing metadata does not match our [spec](https://github.com/googlefonts/gf-docs/blob/master/ProjectChecklist.md#ofltxt). OFL and copyright strings... the usual business.

- Font names do not match our spec as well. Semibold should be SemiBold.

@davelab6 @twardoch What do you folks think, shall we make the +1 crew happy and push it with these errors? I'm certain many fonts in our collection feature such issues. If not, I can only hot fix them in the binaries, since I don't have sources. I really dislike doing this because we are deviating away from the source.  

In the future, if we decide to do an upgrade/language extension for this project. We can fix the issues and convert the sources to .glyphs. For the time being, we could live with the errors, if the authors are happy with this approach and yourself @davelab6.

I will personally do the font splitting so it will work on our api.

ps hinting in Windows looked great. Here's a screenshot of Win 7 Chrome.

![screen shot 2016-11-21 at 08 34 56](https://cloud.githubusercontent.com/assets/7525512/20474235/ef284e46-afc5-11e6-9cb1-d3e0bf07982f.png)

**Comment by VladShcherbin** [comment]
Since current version has this issues and the users are perfectly fine with them, it would be great to have updated version with them as for 99% of users having more families support is the best wish with this update.

**Comment by davelab6** [comment]
On 21 November 2016 at 07:39, Marc Foley <notifications@github.com> wrote:

> Alright,
>
> We have some minor issues and 2 relatively serious ones.
>

Thanks for looking into this Marc!

Lato is like Ubuntu in that its a 'special' family - ultra popular but
without real sources or a reliably automated build process. Adam is the
magician here :)


> Serious FB <https://github.com/googlefonts/fontbakery> report for v2
> binaries <http://www.latofonts.com/lato-free-fonts/#download>
>
> *All Weights*
>
>    - "ERROR: Glyph "uni200C"
>    <http://www.fileformat.info/info/unicode/char/200c/index.htm> has ink.
>    It needs to be replaced by an empty glyph",
>    "ERROR: Glyph "uni200D"
>    <http://unicode.org/cldr/utility/character.jsp?a=200D> has ink. It
>    needs to be replaced by an empty glyph"
>    Lato V2 has:
>    [image: screen shot 2016-11-21 at 08 11 11]
>    <https://cloud.githubusercontent.com/assets/7525512/20473606/1e912fd0-afc2-11e6-9c38-a8a625b31d9b.png>
>
>
That seems bad, but you can just drop the glyphs using pyftsubset (or even
just drop the link from the GLYF table with ttx) to hotfix this :)


>
>    -
>    <https://cloud.githubusercontent.com/assets/7525512/20473606/1e912fd0-afc2-11e6-9c38-a8a625b31d9b.png>
>
> *Some weights*
>
>    - "ERROR: OS/2 usWinAscent & usWinDescent Changes that must be applied
>    to this font: OS/2 usWinAscent from 1974 to 2109 | OS/2 usWinDescent from
>    426 to 527".
>
> This will cause clipping on glyphs which ascend past 1974 units, same with
> the descending glyphs. I Recently found this issue on Overpass
> <https://github.com/RedHatBrand/Overpass/issues/33>. However, the metrics
> do match the previous release. This means it existed in the previous
> version which made it into fonts.google.com
>
If we move this to the "khaled" v metrics schema, where will we see reflow?
Everywhere or only on a subset of platforms (GDI?)


> Minor
>
>    -
>
>    Family needs splitting, due to api font name constraints. Thin and
>    Heavy will need their own separate families.
>
>
I am not sure about this. In http://www.latofonts.com/download/Lato2OFL.zip
there are only 18 font files, so they should be fitting within a single GF
API family.

Where the style names are out of compliance, they can be hotfixed


>
>    - Licensing metadata does not match our spec
>    <https://github.com/googlefonts/gf-docs/blob/master/ProjectChecklist.md#ofltxt>.
>    OFL and copyright strings... the usual business.
>
>
Yeah perhaps this is a good moment to address that since it can be hotfixed


>
>    - Font names do not match our spec as well. Semibold should be
>    SemiBold.
>
>
Ditto


> @davelab6 <https://github.com/davelab6> @twardoch
> <https://github.com/twardoch> What do you folks think, shall we make the
> +1 crew happy and push it with these errors? I'm certain many fonts in our
> collection feature such issues. If not, I can only hot fix them in the
> binaries, since I don't have sources. I really dislike doing this because
> we are deviating away from the source.
>
Knowing your proclivity for automated builds, you would dislike dealing
with the sources even more ;)


> In the future, if we decide to do an upgrade/language extension for this
> project. We can fix the issues and convert the sources to .glyphs. For the
> time being, we could live with the errors, if the authors are happy with
> this approach and yourself @davelab6 <https://github.com/davelab6>.
>

Actually the glyph set of Lato 2.0 is super extensive, so this could be
considered more or less a 'final' release


> I will personally do the font splitting so it will work on our api.
>
> ps hinting in Windows looked great. Here's a screenshot of Win 7 Chrome.
>
> [image: screen shot 2016-11-21 at 08 34 56]
> <https://cloud.githubusercontent.com/assets/7525512/20474235/ef284e46-afc5-11e6-9cb1-d3e0bf07982f.png>
>

I suggest you prep hotfixes for Adam to approve and we ship this 'soon' :)

-- 
Cheers
Dave

**Comment by m4rc1e** [comment]
@davelab6 

Here's my approach to kick this out the door:

- Since Heavy and Hairline weights exist, they will become their own separate families. Our api only support fonts from Thin–Black

- The win asc and desc doesn't bother me for the time being. The fonts already use Khaled's approach, only the win asc and desc are bad. I don't want to tinker with these too much in FontTools. This already existed in the previous versions as well

- I'll change the name Semibold to SemiBold

I think this is a nice temporary solution. It also means I won't spend longer than a few hours getting this done. If we're all happy with this approach,  I'll submit the pr tomorrow :-)

Cheers,
Marc

**Comment by twardoch** [comment]
I don't think the fonts need to become separate families, since there are 9 weights in Lato 2.015 (specifically done so at Dave's request, the old Lato had a slightly different weight progression), so they map well to font-weight 100 to 900. The internal style names have no meaning in webfonts. The webfont kit for self-hosting has no problem with them being in one family. 

The old Lato had 5 weights, and Lato 2.0 has 9 weights, but the design space positions of the previously released 5 weights has been kept (which is why the new Lato Medium is not much bolder than the regular, because both Medium and Semibold needed to fit between the old Regular and the old Bold which were not very far apart in weight). 

I'd very much prefer if the 9 weights remain part of one family. But I realize that the way old Lato has been deployed on GF is that the Thin is 100, while in Lato 2, Hairline is 100 and Thin is 200. 

Also, Lato 2.0 has a bit tighter spacing than Lato 1.0, because we used iKern for the spacing and kerning. But tighter is better than looser, because this won't cause overset text. 

The design changes regarding the two glyphs will need to be tabled for our side. We do have a few issues on our todo list already but they keep sitting there, especially since in September, Łukasz has become a father to twins, and he's currently on paternal leave from work (mostly). 

And yes, the build process is currently complicated and semi-manual. I've been hoping to upgrade it to a more automated process but it has been unobvious which one. `fontmake` is the most promising so I started experimenting with it. But due to the glyphset size, we're currently using some non-opensource tools in the middle of the process and I still haven't found good replacements. :/

**Comment by twardoch** [comment]
Ps. In short, Lato 2 is supposed to be a good replacement for Lato 1, vertical metrics and weights have been kept but the spacing is tighter, so there is no guarantee for full backwards compatibility.

**Comment by davelab6** [comment]
On Nov 21, 2016 8:12 AM, "Marc Foley" <notifications@github.com> wrote:
> Since Heavy and Hairline weights exist, they will become their own
separate families. Our api only support fonts from Thin–Black

Heavy and Hairline must be renamed to Black and Thin.

> The win asc and desc doesn't bother me for the time being. The fonts
already use Khaled's approach, only the win asc and desc are bad. I don't
want to tinker with these too much in FontTools. This already existed in
the previous versions as well

Just fix them

> I'll change the name Semibold to SemiBold

Yep, careful of osx though, you need to rename from x to x1 to X, as
renaming from x to X actually nothing and git goes millennial, err, mental.
Autocorrect 🙈

> I think this is a nice temporary solution. It also means I won't spend
longer than a few hours getting this done. If we're all happy with this
approach, I'll submit the pr tomorrow :-)

I think the above 3 actions don't take very long either 🙊

**Comment by davelab6** [comment]
On Nov 21, 2016 12:56 PM, "Adam Twardoch" <notifications@github.com> wrote:
>
> I don't think the fonts need to become separate families, since there are
9 weights in Lato 2.015 (specifically done so at Dave's request, the old
Lato had a slightly different weight progression), so they map well to
font-weight 100 to 900. The internal style names have no meaning in
webfonts. The webfont kit for self-hosting has no problem with them being
in one family.

Agree

> The old Lato had 5 weights, and Lato 2.0 has 9 weights, but the design
space positions of the previously released 5 weights has been kept (which
is why the new Lato Medium is not much bolder than the regular, because
both Medium and Semibold needed to fit between the old Regular and the old
Bold which were not very far apart in weight).

Agree, and variable fonts will sort this out very soon so I'm not worried
at all about it

> I'd very much prefer if the 9 weights remain part of one family. But I
realize that the way old Lato has been deployed on GF is that the Thin is
100, while in Lato 2, Hairline is 100 and Thin is 200.

I'm ok with this

> Also, Lato 2.0 has a bit tighter spacing than Lato 1.0, because we used
iKern for the spacing and kerning. But tighter is better than looser,
because this won't cause overset text.

I'm also ok with this, and indeed there will be quite a bit of tighter and
looser spacing changes coming across the collection as quality improves.

> The design changes regarding the two glyphs will need to be tabled for
our side.

Why? 😄

> We do have a few issues on our todo list already but they keep sitting
there, especially since in September, Łukasz has become a father to twins,
and he's currently on paternal leave from work (mostly).

🐣🐣😄📣

> And yes, the build process is currently complicated and semi-manual. I've
been hoping to upgrade it to a more automated process but it has been
unobvious which one. fontmake is the most promising so I started
experimenting with it. But due to the glyphset size, we're currently using
some non-opensource tools in the middle of the process and I still haven't
found good replacements. :/

I hope you can speak with the Noto team about this as they also have some
large glyph sets and if Python based fonttools can't cut it, that's
important feedback :)

**Comment by m4rc1e** [comment]
@davelav6 Ok cool.

I may as well write a script so we can do this for other families as well.

Cheers,
Marc

**Comment by m4rc1e** [comment]
Hey folks,

Apologies for the lack of updates on this. I needed to write/update some tools in order to hot fix these fonts. I can confirm the clipping issue and vertical metrics are solved. I adjusted the values but made sure the appearance matches the version which is already hosted.

Win 7 Chrome
![screen shot 2016-11-25 at 09 25 21](https://cloud.githubusercontent.com/assets/7525512/20618818/81dd89bc-b2f1-11e6-8b5c-b4650b3eb4bc.png)

OSX Safari
![screen shot 2016-11-25 at 09 25 47](https://cloud.githubusercontent.com/assets/7525512/20618834/90bbfe32-b2f1-11e6-9cdf-719f40454cd2.png)

OSX Chrome
![screen shot 2016-11-25 at 09 26 41](https://cloud.githubusercontent.com/assets/7525512/20618867/b495abdc-b2f1-11e6-884f-3f9de4b3c3fc.png)


Since I have all the tooling in place, issues and releases such as #10 should happen much quicker. Thank you all for your patience. I still need to submit a few prs to tools I have used, before I can upload the hotfix repo.

@twardoch I'll send you a .zip of the fonts.

Cheers,
Marc

**Comment by m4rc1e** [comment]
Right, I've uploaded the hot fix [repo](https://github.com/googlefonts/LatoGFVersion). Once the authors are happy, I'll submit a pr.

**Comment by twardoch** [comment]
@m4rc1e Do you know why Lato-Light (left) vs. Lato-300-normal-outline (right) has text reflow?

**Comment by twardoch** [comment]
Or — is the left one the new Lato 2.0 while on the right it's old Lato 1.0? (In that case the reflow is justified because Lato 2.0 runs a bit tighter).

**Comment by davelab6** [comment]
@twardoch yep the right is what is live in the Google Fonts API at the moment

**Comment by twardoch** [comment]
OK, go ahead and publish it then.

**Comment by davelab6** [comment]
Thanks Adam!

**Comment by g4b1nagy** [comment]
Love you guys! Thank you for taking the time to update Lato. It means a lato :)
Jokes aside, it seems the changes were merged into [google / fonts](https://github.com/google/fonts) in this [PR](https://github.com/google/fonts/pull/479). Any idea on when we'll be getting the updated version live on [fonts.google.com](https://fonts.google.com/)?
Again, thank you!

**Comment by random-stranger** [comment]
Any update on this, when we'll be happy to use the updated version? 🎉 

cc @davelab6 , @m4rc1e

**Comment by davelab6** [comment]
Sadly I can't say, as its not ready until its ready, but, I hope soon! :)

**Comment by VladShcherbin** [comment]
@davelab6 hey, any news on this?

Is it possible to be released before the end of the year, seems like everything was fixed?

**Comment by davelab6** [comment]
It will be updated in the API soon, but will not be in December

**Comment by haqqi** [comment]
Finding information about when google fonts release Lato 2, and visited this. Waiting this great font to be released in the API.

**Comment by random-stranger** [comment]
@davelab6 Will it ever be released ?

@m4rc1e fixed everything 2 months ago, what are we waiting now? I'm not sure how bad your deploy process is, but this attitude to the people, asking for this font forever, is really bad.

**Comment by eek4ever** [comment]
@davelab6 Hey Men,
we need this font on a Google Fonts!
Please do it ASAP

**Comment by davelab6** [comment]
@m4rc1e was https://twitter.com/janakatino/status/821513539919679488 resolved in https://github.com/google/fonts/pull/479 ?

**Comment by pokonski** [comment]
Hey guys, any news on this?

**Comment by longzheng** [comment]
+1

**Comment by davelab6** [comment]
Waiting on @twardoch

**Comment by willblackmore** [comment]
Hey, @twardoch / @davelab6, I was wondering if there had been any movement on this recently or if there is an ETA?
Thanks!

**Comment by davelab6** [comment]
Adam, any news?

**Comment by twardoch** [comment]
Not yet. But things look promising. Łukasz got back to work after being exclusively committed dad to his newborn twins for some 8 months. There are a handful of glyphs that need tweaking and I think he'll be able to do it soon, then I can make the update with the newest ttfautohint et al. Maybe I can even scrap a big part of my awkward production workflow in favor of fontmake

**Comment by Renkas** [comment]
And another 4 months have passed ...

**Comment by VladShcherbin** [comment]
@Renkas this is normal for google fonts. When you have to wait for an important fix [like this one](https://github.com/google/fonts/issues/588) for half a year, it's no surprise Lato 2.0 is not added for years.

Expect this update in 2020 if you are lucky.

**Comment by ondrek** [comment]
Oh man, I'm waiting to this font to be fixed like 3 years. 
Google ain't an agile company.

**Comment by fitojb** [comment]
And none of you seem to have a remote idea of how long it takes to build (both design and compile) a font, validate it and ensure it doesn’t contain bugs or regressions, a great amount of which is not easy to detect. Have you even tried? Fonts are incredibly complex pieces of software. This is only getting faster these days because of huge investments in automation by Google and other companies, but for many years the tooling didn’t even exist, which was the case when fonts like Lato were created. As these older projects need internal upgrades, that is the bottleneck. And in the meantime, there have been updates to the technical requirements that web fonts need to meet to function properly across browsers, a moving target. So all of this takes time. It’s not like they only have to swap some files on the server. It’s not like Google didn’t try to update the font, if you read this thread (they ran into critical problems with the update).

I’ve always found it funny (and not in a good way) when people complain about things they haven’t paid for. It’ll get fixed when it gets fixed. What stops you from self-hosting?

And please read [this comment](#issuecomment-291512646) again. You’ll be surprised to hear that font authors don’t devote their entire lives to be in front of a computer to please random people on the Internet.

Sorry for being pissed off, but I’m tired of cluttering “+1!” comments on GitHub, where people always demands, but never gives.

(I’m a volunteer contributor, in case you were wondering.)

**Comment by twardoch** [comment]
It's not really Google's fault. 

The current version of Lato 2.x unfortunately has a hinting problem (not evident if you use it for the first time, but visible when your website switches from v1 to v2). 

There was a test deploy of that version on Google Fonts, but we've decided to pull it back, because it affected quite a few people. The build process of Lato is currently complicated (involves various manual actions and custom scripts). There are a few minor glyph set fixes that need to be done as well. I'm incredibly busy at the moment, and it's hard for me to find two or three full days of unpaid time to make the release happen, but I have a plan to work on that. 

In a way, Lato has become a bit of a victim of its own success. Because it's so popular, making changes to it needs extra care so that millions of websites that use it don't suddenly break is some way. 

My plan is to nonetheless find time to release a build that will be good for inclusion on Google Fonts by the end of this year. Please keep your fingers crossed :)

**Comment by VladShcherbin** [comment]
I actually never understood, why major font versions can't be released with corresponding versions:

- lato v1.xx - fonts.google.com/lato.css
- lato v2.xx - fonts.google.com/lato-v2.css

I also don't understand, why you need to make so many fixes.

Users, who want Lato font will download it from Lato font page and use it. They don't care about this fixes, etc - they love font the way it was created. I'm sure, they will happily pick official version hosted on Google Fonts rather than wait another year or more for fixes they don't even care about.

I have no idea, why you need to overcomplicate this and can't just grab the official version and host it on Google Fonts.

**Comment by twardoch** [comment]
Vlad, 

I am the person who has built all the "official versions" of Lato until today. You say: 

> Users, who want Lato font will download it from Lato font page and use it. They don't care about this fixes, etc - they love font the way it was created. I'm sure, they will happily pick official version hosted on Google Fonts rather than wait another year or more for fixes they don't even care about.

You’ve **almost** nailed it. The truth is: 

* Users who don’t care about the fixes that need to be done to Lato v2.0 and are happy with the current state can grab the current 2.0 version from our website, latofonts.com
* People who use Lato on Google Fonts certainly will not want two different versions to be available there (v1 and v2). This will confuse them, and the Google Fonts team (which is just a few people) will be swamped with questions (well, which version should I use?). 
* The Google Fonts team has tried deploying the current version of Lato 2.0 on Google Fonts, and suddenly a huge number of people complained because of the visual rendering problems on Windows. So the Google Fonts team has pulled back that version until I fix these problems.

**Comment by twardoch** [comment]
Ps. There are a number of technical problems with the current version of Lato 2.0. The fact that you haven’t run into them yet doesn’t mean they don’t exist. I know they exist because I get all the feedback from the users who have downloaded the fonts from latofonts.com and used them. If that version went live on Google Fonts, the number of complaints would increase like 100x.

**Comment by VladShcherbin** [comment]
@twardoch one of the best things with google fonts - it's easy to work with: you select fonts, copy-paste code - boom, everything works. It saves you from downloading fonts, uploading, updating, etc.

With multiple versions, there could be a quick summary of what's different (some glyphs, new language sets, etc), even a list of possible bugs to be aware of. I'm sure, if there there was a choice:

- pick a newer version and accept a list of possible bugs
- wait 3 years for a fixed version

a lot of users would choose the first one while the others (who have bugs, afraid of them, etc) would wait or use v1 instead.

At the end of the day, we still download the official version, no matter if there are bugs/issues with it or not. We accept it and use as it is because we like the font and want to use right now. Thank you for your hard work and a great font ❤️

**Comment by twardoch** [comment]
@VladShcherbin — if Lato got two versions (old and new), then other people might want this for other typefaces as well (lots of opensource fonts get revised). As you say, “one of the best things with google fonts - it's easy to work with”. If Google Fonts got “fonts with different revisions” as a feature, I think it would instantly become “less easy to work with”. :)

**Comment by VladShcherbin** [comment]
@twardoch yeah, I'd be happy to see other fonts with revisions.

I think that a lot of users, who plug in fonts from google fonts are already familiar with such things like git, npm, bower, etc. For users, who don't know this - there could be a guide, an article, a youtube video with steps needed.

I'm sure google has resources to create a beta/private version with such new features to get user feedback before launching into the wild. It's a bit disappointing that google fonts team doesn't want to give it a shot. :)

**Comment by twardoch** [comment]
What you’re describing sounds like a wonderful idea for a startup. 

I don’t think that the Google Fonts team “doesn’t want to give it a shot”. Google Fonts is powered by a small dedicated team, and is funded by Google. They need to use their funding wisely. We don’t pay for it, so Google Fonts doesn’t generate any direct revenue for Google. I think there are many wonderful things that the Google Fonts team could or maybe should do, and I can think of a number that would be more pressing. 

Fortunately, I think the users who are familiar with things like git, npm, bower etc. are also familiar with the concept self-hosting of web fonts, in any versions they like. :)

**Comment by mvasilkov** [comment]
Hi all! So can you please tell about the version of Lato hosted on Adobe TypeKit, is that 2.010 as-is? Or did they patch it up in some way for hinting etc.

(I'm just really interested in the Lato v2 font; much less in self-hosting such nuanced things.)

**Comment by twardoch** [comment]
You’d have to ask “them”. I personally have no idea.

**Comment by imaginarny** [comment]
Hey, is there anything new on this issue?

**Comment by WeAreUnique** [comment]
Wow, soo long

**Comment by RichardMisencik** [comment]
4 years and font is still not updated?
Nice job Google

**Comment by m4rc1e** [comment]
Search in the issues bar and read the other threads. You'll discover we rolled out v2.0 and it caused too many regressions so we rolled back, https://github.com/google/fonts/issues/644#issuecomment-278697263

**Comment by RichardMisencik** [comment]
@m4rc1e What about version 2.015?

**Comment by m4rc1e** [comment]
@RichardMisencik that's what we attempted to pr, https://github.com/google/fonts/pull/479/files

**Comment by RichardMisencik** [comment]
@m4rc1e I see, so is there no way Lato v2 will be available on G Fonts?

**Comment by rinart73** [comment]
How could Lato v2 cause any issues if standalone version from official site work perfectly?

**Comment by m4rc1e** [comment]
@rinart73 https://github.com/google/fonts/issues/644

**Comment by emilio-martinez** [comment]
@m4rc1e thanks for mentioning those threads! From my understanding then this is strictly related to a shift from Lato's design, correct? The x- height amongst other details are largely different, so you've opted to remain on v1 to avoid visual regressions?

**Comment by m4rc1e** [comment]
@emilio-martinez almost :-)

There's two issues.

- V2 hinting changed because ttfautohint was updated and it now produces a different result.
- The original designers/engineers are probably busy with other work and personal issues.

**Comment by Cooluck** [comment]
Any news on Lato update on Google Fonts?

**Comment by KiwiBird97** [comment]
This is affecting Canvas LMS on platforms other than macOS. Lato is one of my favorite fonts!

**Comment by krystian-3c** [comment]
in the meantime, is a polyfill possible?
[https://github.com/latofonts/lato-source/issues/6](url)

**Comment by davelab6** [comment]
https://github.com/latofonts/lato-source/#version-3100dev2-2020-08-16

V3.x is nearly ready

@m4rc1e please add to your queue after Roboto and Open Sans

**Comment by roychowdhuryrohit-dev** [comment]
Which Google font supports the character U+20BF (₿)? Any idea?

**Comment by Maia-Everett** [comment]
The Lato font seen on Google Fonts is still missing Cyrillic. Was v2 really added?

**Comment by bublinec** [comment]
I am also experiencing issues with this.

**Comment by adrianvintu** [comment]
Soon will be 10 years. No update?

**Comment by brendon** [comment]
This issue makes for some interesting reading. I was wondering if it was just my imagination, but didn't the Lato font on Google Fonts used to support macrons: āēīōū (specifically used in Māori). I'm sure I remember choosing this font in our web app because of that support, and then earlier this year I noticed that those characters were falling back to the system font. Is my memory faulty?
